src/components/network/equipment-table.js

```diff
@@ -0,0 +1,266 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { requestNetworkChange } from '../../utils/rest-api';
+import { IconButton, TextField } from '@material-ui/core';
+import CreateIcon from '@material-ui/icons/Create';
+import Grid from '@material-ui/core/Grid';
+import CheckIcon from '@material-ui/icons/Check';
+import ClearIcon from '@material-ui/icons/Clear';
+import TableCell from '@material-ui/core/TableCell';
+import LoaderWithOverlay from '../loader-with-overlay';
+import VirtualizedTable from '../util/virtualized-table';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    cell: {
+        display: 'flex',
+        alignItems: 'right',
+        textAlign: 'right',
+        boxSizing: 'border-box',
+        flex: 1,
+        width: '100%',
+        height: '100%',
+        cursor: 'initial',
+    },
+}));
+
+const ROW_HEIGHT = 48;
+
+export const EquipmentTable = ({
```

Comment:
Is it better to only pass the array of the ressource instead of the full network ?

Reply:
need network to fetch the equipment,